frontend/src/components/top-navbar.tsx

```diff
@@ -25,7 +26,7 @@ export const TopNavBar: FCWOC<{ className?: string }> = ({ children, className }
                             <Menu alignEnd >
                                 {isMobile && <li><a className="dropdown-item" href="/studies">Studies</a></li>}
                                 <li><a className="dropdown-item" href="/account">My account</a></li>
-                                <li><a className="dropdown-item" href={env.logoutURL}>Log out</a></li>
+                                <li><a className="dropdown-item" href={env.logoutURL} onClick={user.logout}>Log out</a></li>
```

Comment:
currently logging out in the model is a "dev env only" function.  

When deployed to staging/prod, the BE doesn't even have that controller loaded.   Also calling the `logout` function directly in the onClick will lose the user class scope and any calls to `this` inside the method will fail.  It'd need to be something like: `onClick={() => user.logout()}` or have the method bound to "this" in order to work.  I'm guessing you're not seeing that because it's loading the logoutURL and console error messages are lost.

How about changing the user method (https://github.com/openstax/kinetic/blob/main/frontend/src/models/user.ts#L59) to something like:
```ts
    logout: async (ev?: React.MouseEvent<HTMLAnchorElement>) => {
        ev?.preventDefault()
        if (ENV.IS_DEV_MODE) {
            await fetch(`${ENV.API_ADDRESS}/development/users/log_out`, {
                method: 'DELETE', credentials: 'include',
            })
            this.id = ''
            this.isAdmin = this.isResearcher = false
            window.location = '/users' // the dev user picker screen, not sure this is it
        } else {
            window.location = env.logoutURL
        }
    }
```

That probably has typo's and such, but you get the idea 😁